src/lib/y2partitioner/widgets/format_and_mount.rb

```diff
@@ -287,7 +293,57 @@ def validate
         false
       end
 
-    private
+      # Check if a system mount point is reused without formatting the
+      # partition during installation and warn the user if it is.
+      #
+      # @return [Boolean] true if okay, false if not
+      def validate_system_mount_points
+        return true unless Yast::Mode.installation
+        return true if to_be_formatted?
+        return true if !mounted?
```

Comment:
NP: This is one of those cases in which "unless" really helps to save CPU cycles in my brain. :-)